src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c

```diff
@@ -663,7 +660,7 @@ void EnPoSisters_SetupDeathStage1(EnPoSisters* this) {
     this->actor.world.pos.y += 42.0f;
     this->actor.shape.yOffset = -6000.0f;
     this->actor.flags &= ~ACTOR_FLAG_TARGETABLE;
-    this->poSisterFlags = POE_SISTERS_FLAG_CLEAR;
+    this->poSisterFlags = 0;
```

Comment:
Yeah I guess its fine. Feels off to me to see clear not get a macro or define just because every other use of the flags is modded for readibilty, even if its obvious at a glance what it's doing.